server/views/formPages/categoriser/womensProvisionalCategory.html

```diff
@@ -0,0 +1,118 @@
+{% extends "../../formPages/formTemplate.html" %}
+
+{% set pageTitle = "Provisional categorisation" %}
+
+{% from "warning-text/macro.njk" import govukWarningText %}
+{% from "radios/macro.njk" import govukRadios %}
+{% from "textarea/macro.njk" import govukTextarea %}
+{% from "button/macro.njk" import govukButton %}
+{% from "details/macro.njk" import govukDetails %}
+
+{% block navigation %}
+
+  {% include "../../partials/breadCrumbTaskList.html" %}
+
+{% endblock %}
+
+{% set cat = data.suggestedCat %}
+{% set greenButtonText = "Submit" %}
+
+{% set warningCategoryOpenConditions %}
+  {% set categoryOpenConditionsText = "Making this category change means that the categoriser will have to provide more information." %}
+  {% include "../../partials/warningIndeterminate.html" %}
+  {{ govukWarningText({
+    text: categoryOpenConditionsText,
+    iconFallbackText: "Warning",
+    id: "openConditionsInfoMessage",
+    classes: "forms-warning-text warning-overrides background-white"
+   }) }}
+
+{% endset %}
+
+{% set overriddenCategoryTextHtml %}
+  {{ govukTextarea({
+    name: "overriddenCategoryText",
+    type: "myTextWhatsThisFor",
+    id: "overriddenCategoryText",
+    attributes: { 'maxlength': 50000 },
+    label: {
+      text: "Explain why"
+    },
+    hint: {
+      text: "Your comments will be disclosed to the prisoner"
+    },
+    value: data.categoriser.provisionalCategory.overriddenCategoryText,
+    errorMessage: {
+      text: "Enter the reason why you changed the category"
+    } if (errors | findError('overriddenCategoryText'))
+  }) }}
+{% endset %}
+
+{% block formItems %}
+
+<h1 class="govuk-heading-l">Provisional category</h1>
+
+<div class="govuk-warning-text forms-info-text warning-overrides">
+  <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
+  <strong class="govuk-warning-text__text">
+    <span class="govuk-warning-text__assistive">Warning</span>
+    Based on the information provided, the provisional category is {{ catDisplay(cat) }}
```

Comment:
Will be always closed

Reply:
What I meant is, this text will always be 'The provisional category is closed', as per the screenshot on the ticket.